validator/src/main/resources/expected-data-template/otelSDKexpectedHTTPTrace.mustache

```diff
@@ -12,29 +10,26 @@
       "status": 200
     }
   },
+  "metadata": {
+    "default": {
+      "otel.resource.telemetry.sdk.name": "opentelemetry"
+    }
+  },
   "subsegments": [
     {
+      "name": "HTTP GET",
       "fault": false,
       "error": false,
-      "throttle": false,
-      "subsegments": [
-        {
-          "name": "aws.amazon.com",
-          "fault": false,
-          "error": false,
-          "throttle": false,
-          "http": {
-            "request": {
-              "url": "https://aws.amazon.com/",
-              "method": "GET"
-            },
-            "response": {
-              "status": 200
```

Comment:
@wyTrivail The test framework was already validating status code 200 before, this PR does not change that, so I think it's out of scope for the changes here.

Reply:
i suspect this template would make the test unstable even though it's already there. in collector test, we use the spark one, as you can see there we removed lots of items...

Reply:
For JS one the current expected template will not work due to some minor issues such as status code/names. I think the question is should we simplify it like the Spark one so that it can work for JS as well and be more stable?

Reply:
I think we can remove all the names for now but I believe status code should be `200`

Reply:
i think it might take some time for us to find the balance between complexity and stability.  i'm okay with the current the template since we can improve it later. we don't need make it perfect at the beginning. 

in additional, we should always try our best to add unit tests to validate the data structure instead of relying on end to end test, which is consuming multi resources and hard to debug. 

for the js sdk one, maybe you can try to use the spark template, let me know if there's any field js sdk can't support.

Reply:
To keep the validation on `HTTP GET` for the span name (which I think is useful), we could merge this PR for now and then create a new one for JavaScript later. We should create an issue in the JavaScript repo that includes @KKelvinLo 's concerns and link it to that PR to remove the custom JavaScript template when the fixes are included.

Reply:
@wyTrivail I think that's good justification, we can always improve it later and this does look good for now.